contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,50 @@
+# The Benefits of Exercis(ing your Options)
```

Comment:
```suggestion
# A complete and simple guide to options at PostHog
```

I think the current title isn't great - the benefits of exercising = making money. This guide is much more extensive